.prettierrc.json

```diff
@@ -2,5 +2,14 @@
     "singleQuote": true,
     "trailingComma": "all",
     "tabWidth": 4,
-    "organizeImportsSkipDestructiveCodeActions": true
+    "organizeImportsSkipDestructiveCodeActions": true,
+    "overrides": [
+        {
+            "files": ["*.mdx", "*.md", "*.yml", "*.yaml"],
```

Comment:
Prettier doesn't currently allow overriding options for fenced or multi-language contexts (e.g. code blocks in markdown) since overrides are strictly per-file-glob.

In this case, we want to share these overrides for both markdown and yaml, so it works in our favor.